test/functional/wallet_fundrawtransaction.py

```diff
@@ -599,8 +601,12 @@ def test_locked_wallet(self):
 
         # Choose input
         inputs = wallet.listunspent()
-        # Deduce fee to produce a changeless transaction
-        value = inputs[0]["amount"] - Decimal("0.00002200")
+
+        # Deduce exact fee to produce a changeless transaction
+        expected_tx_size = 110  # Total tx size: 110 vbytes, p2wpkh -> p2wpkh. Input 68 vbytes + rest of tx is 42 vbytes.
+        fee_rate = self.min_relay_tx_fee / 1000  # btc/vbytes
+        value = inputs[0]["amount"] - (fee_rate * expected_tx_size)
```

Comment:
nit: could use the `get_fee` helper (from test_framework.util) for not having to bother with manual fee-rate unit coversion and fee calculation:
```suggestion
        value = inputs[0]["amount"] - get_fee(expected_tx_size, self.min_relay_tx_fee)
```

Reply:
sure, pushed. Thanks.
would be nice to have a document somewhere mentioning the existence of these utility functions.